201-nsg-dmz-in-vnet/Readme.md

```diff
@@ -2,4 +2,7 @@ This template creates a VNet with 3 Subnets, Frontend, App Subnet and Database s
 
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-nsg-dmz-in-vnet%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-nsg-dmz-in-vnet%2Fazuredeploy.json" target="_blank">
```

Comment:
I noticed VNET is called VPN on the tool. Is there a reason? Seems to cause confusion when it's called VNET in JSON and VPN in visual